app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -760,20 +771,17 @@ public void onAnimationEnd(Animator animation) {
 
     @Override
     protected boolean onToolbarLongClicked(View v) {
-        if (getUserVisibleHint()) {
```

Comment:
getUserVisibleHint is depreciated.

Reply:
maybe Fragment::isResumed is relevant?  
 <https://stackoverflow.com/questions/57885849/in-androidx-fragment-app-fragment-setuservisiblehint-is-deprecated-and-not-exec/68987518#68987518>